packages/sdk/src/lib/dataProtectorSharing/subgraph/getRentalsQuery.ts

```diff
@@ -0,0 +1,45 @@
+import { gql } from 'graphql-request';
+import { throwIfMissing } from '../../../utils/validators.js';
+import type { GetRentalsGraphQLResponse } from '../../types/graphQLTypes.js';
+import type { GetRentalsParams } from '../../types/index.js';
+import type { SubgraphConsumer } from '../../types/internalTypes.js';
+
+export const getRentalsQuery = async ({
+  graphQLClient = throwIfMissing(),
+  renterAddress,
+  protectedDataAddress,
+  includePastRentals = false,
+}: SubgraphConsumer & GetRentalsParams): Promise<GetRentalsGraphQLResponse> => {
+  const rentalsQuery = gql`
+    query ($where: Rental_filter) {
+      rentals(where: $where) {
```

Comment:
Simpler "where" here in query.
Conditional filters are applied in the "variables" down below. I found it cleaner than conditions in this template string (like in `getProtectedDataInCollectionsQuery.ts` where it's... not that clean)